### PR TITLE
Don't require spaces before function parens

### PR DESCRIPTION
**Pull request by pmg103**
As this is not canonical ES6 style https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -101,7 +100,8 @@
     "es6": true
   },
   "ecmaFeatures": {
-    "jsx": true
+    "jsx": true,
+    "modules": true
```
Comment (JakeSidSmith) [inline .eslintrc]: This not already get merged? (wat)

Comment (pmg103) [inline .eslintrc]: Yeah it has been. GH is talking :hankey:

Comment (JakeSidSmith) [inline .eslintrc]: Kk. XD

**Comment by JakeSidSmith** [comment]
Change from 2 to 0, and you've got a :sparkles:

**Pull request merged**